test/unit/test-amp-story-player.js

```diff
@@ -138,4 +158,36 @@ describes.realWin('AmpStoryPlayer', {amp: false}, env => {
       expect(stories[3][IFRAME_IDX]).to.eql(undefined);
     }
   );
+
+  it('should register handlers at build time', async () => {
+    buildStoryPlayer();
+    const player = new AmpStoryPlayer(win, playerEl);
+
+    env.sandbox
```

Comment:
If you think these lines will be repeated in every single test, should we define a helper that creates the player + stubs `initializeHandshake_`?